CRM/ManageLetterheads/Hook/BuildForm/AddLetterheadSelect.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/**
+ * Adds the letterhead select element to the Email and PDF Letter forms.
+ *
+ * The list of letterheads that should be visible for each form is also appended
+ * as a configuration variable.
+ */
+class CRM_ManageLetterheads_Hook_BuildForm_AddLetterheadSelect {
```

Comment:
This class name looks like it's adding a select field for the letterhead on the PDF and Email forms but looks like it's not doing that, rather adding some config for use in FE. Can we find a better name for it then?

Reply:
By adding this hook we expose the list of letterheads to the FE, but also append the script that creates the dropdown.

Do you recommend a specific name for this?

Reply:
It's fine in that case then